### PR TITLE
Feature/update db api/users/antonio

### PR DESCRIPTION
**Pull request by JAntonioP10**
Refactor validations for checkin, and update the methods and unit tests

### CODE REVIEW

File: src/API/users/checkin.js [hunk 1]
```diff
@@ -1,70 +1,73 @@
...
+const logger = require("../../log/logger");
+const checkin = require("../../data/schema/checkin-schema");
+const users = require("../../data/schema/user-schema");
+const moment = require("moment");
+const { validationResult } = require("express-validator");
 
 module.exports = function(req, res) {
+  const errores = validationResult(req);
+  if (!errores.isEmpty())
+    return res.status(422).json({ errores: errores.array() });
```
Comment (DellChuy) [inline src/API/users/checkin.js]: We should use only english I will suggest use "error" instead of "errores"

File: src/API/users/PTO.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+const logger = require("../../log/logger");
+const checkin = require("../../data/schema/checkin-schema");
+const users = require("../../data/schema/user-schema");
+const moment = require("moment");
+const { validationResult } = require("express-validator");
+
+module.exports = function(req, res) {
+  const errores = validationResult(req);
+  if (!errores.isEmpty())
+    return res.status(422).json({ errores: errores.array() });
```
Comment (Miguelsiqueiros) [inline src/API/users/PTO.js]: We need to change this to english.

File: src/API/users/checkin.js (hunk 1 above)
Comment (Miguelsiqueiros) [reply src/API/users/checkin.js]: done

File: src/API/users/register.js [hunk 3]
```diff
@@ -1,24 +1,33 @@
...
 
-module.exports = function(req, res) {
+module.exports = async function(req, res) {
   const { name, image, email } = req.body;
   const errores = validationResult(req);
   if (!errores.isEmpty())
     return res.status(422).json({ errores: errores.array() });
 
+  if (await users.findOne({ email: email }))
+    return res.status(409).json({ errores: "Email already in use." });
```
Comment (DellChuy) [inline src/API/users/register.js]: English please!

**Pull request merged**